contracts/Vault.sol

```diff
@@ -53,7 +54,7 @@ contract Vault is IVault, Initializable {
   }
 
   /// @inheritdoc IVault
-  function updatePeriod() external returns (uint256 _period) {
+  function emissionReward() external returns (uint256 _period) {
```

Comment:
emission is a noun, emit is its verb form

Reply:
ok